web-registry/src/components/organisms/BridgedEcocreditsTable/BridgedEcocreditsTable.constants.ts

```diff
@@ -1,6 +1,9 @@
 export const BRIDGED_STATUS = ['pending', 'complete', 'error'];
+
 export const CREDIT_BATCH_TOOLTIP =
   'Each credit batch has a unique ID (i.e. denomination) that starts with the abbreviation of the credit type followed by the project ID, start date and end date of the vintage year, and ending in the batch sequence number.';
 export const NO_BRIDGED_CREDITS = 'No bridged ecocredits found';
 export const AMOUNT_BRIDGED_TOOLTIP =
   'Amount bridged is the same as amount cancelled in the ledger documentation';
+
+export const DATE_FORMAT_STRING = 'MMM D, YYYY';
```

Comment:
this is probably fine, but seems like we should have a `DATES` constants somewhere to centralize the formats we use. Otherwise we’re using constants, but aren’t necessarily staying very DRY

Reply:
Done, thanks @haveanicedavid ! I've moved it along with the function that consumes it, in the format file in web-components utils. Nice!